FinancialTracker/ViewControllers/BalanceViewController.swift

```diff
@@ -18,12 +18,15 @@ class BalanceViewController: UIViewController {
         super.viewDidLoad()
         balanceTextField.keyboardType = .numberPad
         
+        // Data is fetched too slow and this crashes. Will be used in the future
```

Comment:
Your comment is not entirely correct here because &#8595; 
![image](https://user-images.githubusercontent.com/18184558/147549746-f5a1ec87-6609-481e-9f5b-d11e2a777315.png)